src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/CreateVariantIngestFiles.java

```diff
@@ -186,11 +187,14 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
             return;
         }
 
-
-        // create VET output
-        if (!variant.isReferenceBlock()) {
+        // write to VET if NOT reference block and NOT a no call
+        // getGenotypes() returns list of lists for all samples at variant
+        // assuming one sample per gvcf, getGenotype(0) retrieves GT for sample at index 0
+        final int SAMPLE_INDEX_NOCALL_CHECK = 0;
```

Comment:
just style... I think it's fine to not have this pulled out into a variable... but if you do
1. all caps typically indicates a static constant (so I would move this up and do that)
2. you do the same getGenotype(0) in PetTsvCreator, so it should refer to the same constant
3. at that point you might as well pull this into a static method like isNoCall(VariantContext v) can call that where you need it.

BUT -- I would say just doing the call to getGenotype(0) with the above comment is totally appropriate, just right now it's sort of halfway between the two options

Reply:
i liked option 3 so i gave it a go, think that it works and i did it correctly since testing all checks out but if i could get eyes on if i organized that method/code physically in the right place within the structure of the code that would be amazing.